backend/tests/test_albums.py

```
@@ -318,13 +339,16 @@ def test_get_album_by_id_not_found(self):
-                    "Secret",
-                    1,
-                    bcrypt.hashpw("oldpass".encode(), bcrypt.gensalt()).decode(),
-                    0,
+                album_row(
+                    {
+                        "album_id": "abc-456",
+                        "album_name": "Locked Album",
+                        "description": "Secret",
+                        "is_locked": True,
+                        "password_hash": bcrypt.hashpw(
+                            "oldpass".encode(), bcrypt.gensalt()
```

Ruff (0.16.0)

[warning] Unnecessary call to `encode` as UTF-8

Rewrite as bytes literal

(UP012)